content/management/index.mdx

```diff
@@ -8,20 +8,127 @@ desc: "There are dedicated APIs for the resource management. Use the Configurati
+
+- **developer preview (dev preview)**
+
+  The unstable version that provides a preview of the upcoming functionalities. **It's not open for public use by default.** If you want to use it, contact us at developers@livechat.com or <a href="#open-chat" onClick={openChatWindow}>ask on the chat</a>, and we'll give you access.
+
+- **legacy**
+
+  The old version we still support. We highly encourage you to switch to the latest stable version.
+
+**Currently:**
+
+|                 | Configuration API |
+| --------------- | ----------------- |
+| **stable**      | 3.3               |
+| **dev preview** | 3.4               |
+| **legacy**      | 2.0, 3.1, 3.2     |
+
+<Warning>
+
+Currently, Configuration API v2 and v3 cover different resources. However, if you're still using v2, we encourage you to switch to v3 as more and more resources are being migrated to v3.
+
+</Warning>
+
+# Resources
+
+## Agents
+
+An agent is a type of user who communicates with customers. You can see the sample agent data structure in the [response of Get Agent](/management/configuration-api/#get-agent). The Configuration API allows for multiple agent-related actions, including creating, updating, deleting, and suspending agents.
+
+<SectionLink to={"/management/configuration-api/#agents"}>
+  See agent methods
+</SectionLink>
+
+## Auto access
+
+Auto access is a functionality that allows for assigning customers to dedicated groups based on the URL or the customer's geolocation. When a customer who matches some pre-defined auto access rules starts a chat, the chat is automatically routed to the specific group. You can use auto access, for example, to provide multi-language support with each group of agents being responsible for communication in a different language.
+
+<SectionLink to={"/management/configuration-api/#auto-access"}>
+  See auto access methods
+</SectionLink>
+
+## Bots
+
+Bots are similar to regular agents. The main difference between them is that bots have a greater potental for automation, for example, they can listen for and react to incoming webhooks and pushes.
+
+Bots are created and managed via the Configuration API. Just as regualar agents, bots can call Agent Chat API by the Web or RTM API.
+
+Bots are authorized with the use of the [agent token](/authorization/authorizing-api-calls/#agent-authorization-flows). Using bots requires sending the `X-Author-Id` header for the Web API and the `author_id` property for the RTM API.
+
+To change a bot's status (`accepting_chats`, `not_accepting_chats`, or `offline`), use the [Set Routing Status](/messaging/agent-chat-api/#set-routing-status) method from the Agent Chat API.
+
+<SectionLink to={"s/management/configuration-api/#bots"}>
+  See bot methods
+</SectionLink>
+
+## Groups
+
+Groups let you organize your work by creating teams with agent and bot members. You can use groups to do configuration based on shared settings (language, working hours, department) and to separate chat routing.
```

Comment:
This one seems a little short compared to the other ones, maybe we can add a note similar to the one we added for the agents?

```suggestion
Groups let you organize your work by creating teams with agent and bot members. You can use groups to do configuration based on shared settings (language, working hours, department) and to separate chat routing. With the Configuration API, you can perform actions such as creating and deleting a group, or updating its' settings.
```